openshift/datasync-showcase-amq.yml

```diff
@@ -263,21 +227,17 @@ objects:
           volumeMounts:
           - mountPath: /usr/src/app/files
             name: files-storage
+          - mountPath: /usr/src/app/website/assets/config
+            name: webapp-config
         volumes:
         - name: files-storage
           persistentVolumeClaim:
             claimName: sync-files-storage
+        - configMap:
+          name: webapp-config
```

Comment:
If you compare this with the docs: https://docs.openshift.com/container-platform/3.11/dev_guide/configmaps.html#configmaps-use-case-consuming-in-volumes

It's possible that whatever parses the templates is getting confused because you're missing a top level name variable. I would try changing it to this.

```
- name: webapp-config
  configMap:
    name: webapp-config
```